src/components/OSCALCatalogGroups.js

```diff
@@ -0,0 +1,96 @@
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid";
+import List from "@mui/material/List";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import { styled } from "@mui/material/styles";
+import Tab from "@mui/material/Tab";
+import Tabs from "@mui/material/Tabs";
+import PropTypes from "prop-types";
+import React from "react";
+import { OSCALSection, OSCALSectionHeader } from "../styles/CommonPageStyles";
+import OSCALCatalogGroup from "./OSCALCatalogGroup";
+
+export const OSCALControlList = styled(List)`
+  padding-left: 2em;
+  padding-right: 2em;
+`;
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`vertical-tabpanel-${index}`}
+      aria-labelledby={`vertical-tab-${index}`}
+      {...other}
+    >
+      {value === index && <Box sx={{ p: 3 }}>{children}</Box>}
+    </div>
+  );
+}
+
+const ComponentTabs = styled(Tabs)(
+  ({ theme }) => `
+  border-right: 1px solid ${theme.palette.divider};
+`
+);
+
+function a11yProps(index) {
+  return {
+    id: `vertical-tab-${index}`,
+    "aria-controls": `vertical-tabpanel-${index}`,
+  };
+}
+
+TabPanel.propTypes = {
+  index: PropTypes.number.isRequired,
+  value: PropTypes.number.isRequired,
+};
+
+export default function OSCALCatalogGroups(props) {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <OSCALSection>
+      <Card>
+        <CardContent>
+          <Grid container>
+            <Grid item sm={12}>
+              <OSCALSectionHeader>Control Groups</OSCALSectionHeader>
+            </Grid>
+            <Grid item sm={2}>
+              <ComponentTabs
+                onChange={handleChange}
+                orientation="vertical"
+                variant="scrollable"
+                value={value}
+              >
+                {props.groups?.map((group, index) => (
+                  <Tab
+                    key={group.title}
+                    label={group.title}
+                    {...a11yProps(index)}
+                  />
+                ))}
+              </ComponentTabs>
+            </Grid>
+            <Grid item sm={10}>
+              {props.groups?.map((group, index) => (
+                <TabPanel key={group.title} value={value} index={index}>
```

Comment:
This is a place where we use `group.title` for key. Let's be consistent around the `key` we use for groups